app/lib/capital_subtotals.rb

```diff
@@ -6,6 +6,22 @@ def blank
           capital_contribution: 0,
           combined_assessed_capital: 0)
     end
+
+    def unassessed(applicant:, partner:)
+      new(
+        applicant_capital_subtotals: PersonCapitalSubtotals.unassessed(disputed_vehicles: unassessed_vehicles(applicant, disputed: true), non_disputed_vehicles: unassessed_vehicles(applicant, disputed: false), properties: []),
```

Comment:
this feels slightly strange - we have a properties parameter, but we're setting it to [] here rather than letting subtotals do it? (P.S. There is an almost identical bug involving properties!)